src/containerStyles.js

```diff
@@ -1,75 +1,100 @@
-exports.wrapperStyles = (props) => {
-  return {
-    background: props.backgroundColor,
-    borderRadius: '10px',
-    border: `1px solid ${props.foregroundColor}`,
-    height: '30px',
-    opacity: 0.6,
-    padding: "5px 0 5px 10px",
-    position: 'absolute',
-    right: '10px',
-    top: '5px',
-    width: '240px',
-    zIndex: '9999',
-  };
-};
-
-exports.inputStyles = (props) => {
-  return {
-    background: "none",
-    border: "none",
-    color: props.foregroundColor,
-    fontSize: '0.8em',
-    opacity: 1,
-    outline: "none",
-  };
-};
+const { STYLE_CLASSES } = require('./constants');
 
-const buttonStyles = (props) => {
-  return {
-    background: props.foregroundColor,
-    border: 'none',
-    color: props.backgroundColor,
-    height: '100%',
-    paddingBottom: '2px',
-    opacity: 0.6,
-    outline: 'none',
-    position: 'relative',
-    top: '-2px',
-    width: '12%',
-  }
+const buttonStyles = (config) => {
+  return `
+    background: ${config.foregroundColor};
+    border: none;
+    color: ${config.backgroundColor};
+    height: 100%;
+    padding-bottom: 2px;
+    opacity: 0.6;
+    outline: none;
+    position: relative;
+    top: -2px;
+    width: 12%;
+  `
 }
 
-exports.caseButtonOffStyles = (props) => {
-  return Object.assign({}, buttonStyles(props), {
-    borderRadius: '12px 12px 12px 12px',
-  });
-};
 
-exports.caseButtonStyles = (props) => {
-  return {
-    background: props.backgroundColor,
-    border: 'none',
-    color: props.foregroundColor,
-    height: '100%',
-    paddingBottom: '2px',
-    opacity: 0.6,
-    outline: 'none',
-    position: 'relative',
-    top: '-2px',
-    width: '12%',
-    borderRadius: '12px 12px 12px 12px',
-  }
-};
+exports.decorateConfig = (config) => {
+  const hyperSearchUI = Object.assign({
+    inputBorderRadius: 10,
+    buttonBorderRadius: 10,
+    prevButton: '◀',
+    nextButton: '▶',
```

Comment:
could u please add a little description about these two to the readme file? (prev and nextButton, )